README.md

```diff
@@ -8,6 +8,14 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `yarn run confd:prod`
```

Comment:
Please also add documentation about the command:
yarn run confd:prod:indocker

Reply:
basically this one never will be used directly.
I add it just for us (internally)

Reply:
I add new relevant section in **Readme.md**